contrib/board_files/Makefile.am

```diff
@@ -1 +1,29 @@
-sysconf_DATA = @board@/libsoc_gpio.conf
+BOARD_FILES = $(shell find ./ -name \*.conf)
+
+install-data-local:
+	for p in $(BOARD_FILES); do \
+		echo $(MKDIR_P) "$(DESTDIR)$(pkgdatadir)/$$(dirname $$p)";\
+		$(MKDIR_P) "$(DESTDIR)$(pkgdatadir)/$$(dirname $$p)";\
+		echo $(INSTALL_DATA) $$p "$(DESTDIR)$(pkgdatadir)/$$(dirname $$p)/" ;\
+		$(INSTALL_DATA) $$p "$(DESTDIR)$(pkgdatadir)/$$(dirname $$p)/" || exit $$?;\
```

Comment:
then set -x at the end, save forgetting to update the echo and see what is actually happening